### PR TITLE
Bug: scheduler hanging compute resources, doc security context

### PR DESCRIPTION
**Pull request by IceKhan13**
### Summary

Bug: scheduler hanging compute resources, doc security context

### CODE REVIEW

File: gateway/api/schedule.py [hunk 1]
```diff
@@ -48,10 +48,17 @@ def execute_job(job: Job) -> Job:
...
-        job.compute_resource = authors_resource
-        job = submit_ray_job(job)
-        job.status = Job.PENDING
-        job.save()
+        try:
+            job.compute_resource = authors_resource
+            job = submit_ray_job(job)
+            job.status = Job.PENDING
+            job.save()
+        except Exception:
```
Comment (akihikokuroda) [inline gateway/api/schedule.py:56]: You may need `# pylint: disable=broad-exception-caught`

**Review by akihikokuroda — APPROVED** [review]
LGTM

**Comment by psschwei** [comment]
We need this one and #717 right?

**Comment by IceKhan13** [comment]
yes

**Review by psschwei — APPROVED** [review]

**Pull request merged**